documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -2,40 +2,177 @@
 [id='proc-generating-reassignment-json-files-{context}']
 = Generating reassignment JSON files
```

Comment:
@scholzj - do we want two procedures with and without TLS? Or shall we say assume that secure is the standard approach and mention that you won't need the TLS config properties if you don't want to use TLS?

Reply:
I don't know ... the more things we have the harder the maintenance. I would probably stick with just one example.

Reply:
Okay. I've left as:

> The steps describe a secure reassignment process that uses TLS.
> You'll need a Kafka cluster that uses TLS encryption and authentication.